tests/cross_fw/test_templates/template_test_nncf_tensor.py

```diff
@@ -158,7 +164,7 @@ def test_comparison_tensor(self, op_name):
         res = fn(tensor_a, tensor_b)
         res_nncf = fn(nncf_tensor_a, nncf_tensor_b)
 
-        assert res == res_nncf
+        assert res_nncf == res
```

Comment:
Could you clarify why you changed the order here?

Reply:
with the old order comparison operator for `tf.Tensor` was called here, it tries to convert `nncf.Tensor` to `tf.EagerTensor`. It causes a problem because it calls `__len__` operator somewhere inside. I tried to implement `__len__` operator for `nncf.Tensor` as `return len(self._data)` but the comparison still fails for scalar tensors. currently I have no idea how to resolve it and used changing order here as a quick workaround. I'd appreciate any other suggestions.